swap/src/protocol.rs

```diff
@@ -80,6 +82,15 @@ pub enum State {
     Bob(BobState),
 }
 
+impl Display for State {
```

Comment:
This is not needed anymore because we're not computing the state name in `database.rs` anymore.